src/platform/packages/shared/kbn-esql-ast/src/commands_registry/commands/from/autocomplete.ts

```diff
@@ -18,7 +18,17 @@ import { getRecommendedQueriesSuggestions } from '../../options/recommended_quer
 import { withinQuotes } from '../../../definitions/utils/autocomplete/helpers';
 import type { ICommandCallbacks } from '../../types';
 import { type ISuggestionItem, type ICommandContext } from '../../types';
-import { getOverlapRange, isRestartingExpression } from '../../../definitions/utils/shared';
+import {
+  endsWithSpace,
```

Comment:
nit: At this point is only used here so it doesnt make a lot of sense to be on the shared folder